app/controllers/downloads_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DownloadsController < ApplicationController
```

Comment:
We need this so that the general public can access the assets when searching.  As I said in the PR comment, I have no idea why this wasn't failing before now.